.github/workflows/pr.yaml

```diff
@@ -39,6 +39,9 @@ jobs:
       all-linters: ${{ steps.post-filter.outputs.out }}
       # shortened paths to linter subdirs
       linters-files: ${{ steps.post-filter-paths.outputs.out }}
```

Comment:
Can we add nightly tests too? Can be its own job, doesn't need to be integrated with the rest rn, but I'd like to start having some viz on this